cfgov/unprocessed/css/molecules/pagination.less

```diff
@@ -0,0 +1,139 @@
+/* topdoc
+  name: Pagination
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+          <nav class="m-pagination">
+              <a class="btn btn__super m-pagination_btn_prev" href="#o-pagination-content_list">
+                  <span class="btn_icon__left cf-icon cf-icon-left"></span>
+                  Previous
+              </a>
+              <a class="btn btn__super m-pagination_btn_next" href="#o-pagination-content_list">
+                  Next
+                  <span class="btn_icon__right cf-icon cf-icon-right"></span>
+              </a>
+              <form data-js-hook="validate_pagination" action="#o-pagination-content_list">
+                  <label for="m-pagination_current-page">
+                      Page
+                      <span class="u-visually-hidden">
+                          number out of {{ value.pages }} total pages
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/e2dacda37545a885a64b835357988642043605f8